CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Remove new line ([#2681](https://github.com/alphagov/govuk_publishing_components/pull/2681))
```

Comment:
I don't think this needs a CHANGELOG entry.

Reply:
Fair - I've removed